sensorlib/src/CarlaCDASimAPI.py

```diff
@@ -94,13 +94,17 @@ def create_simulated_semantic_lidar_sensor(self, simulated_sensor_config, carla_
         :param sensor_rotation: Sensor rotation in degrees.
         :return: A registered SimulatedSensor.
         """
-
+        is_return = False
         # Parameter checks
-        if not isinstance(infrastructure_id, int) or infrastructure_id < 0:
-            print("Error: infrastructure_id needs to be a non-negative integer.")
-            return None
-        if not isinstance(sensor_id, int) or sensor_id < 0:
-            print("Error: sensor_id needs to be a non-negative integer.")
+        if not isinstance(infrastructure_id, str):
+            print("Error: infrastructure_id needs to be a string.")
```

Comment:
Have you confirmed with the test file that it now works when we pass an object with a string infrastructure ID and sensor id

Reply:
yes, tested and no more errors we see the other day

Reply:
On client and server side